README.md

```diff
@@ -275,6 +279,41 @@ test. The Terraform files then allow us to build what the resource structure is
 expected to look like and once the tool parses the API response, we can compare
 that to the static file.
 
+Suggested local testing steps:
+
+1. Create a main file (do not commit this up)
+
+      ```bash
+      cat > main.tf <<EOF
+                  terraform {
+                    required_providers {
+                      cloudflare = {
+                        source = "cloudflare/cloudflare"
+                        version = "~> 4"
+                      }
+                    }
+                  }
+                  EOF
+      ```
+
+2. Initialize terraform
+
+      ```bash
+      terraform init
+      ```  
+
+3. Run tests (Cloudflare Install path should be path to repository)
```

Comment:
if you initiliase `terraform` in the same directory as `cf-terraforming`, this shouldn't be needed as it defaults to `.`.

Reply:
I actually tried this an it still failed. Potentially its not set to the root directory of the project. I am going to just add it to the make file and we will see if it causes issues or if you don't like that solution.